mypy/semanal.py

```diff
@@ -2128,6 +2154,7 @@ def add_field(var: Var, is_initialized_in_class: bool = False,
         add_field(Var('_field_types', dictype), is_initialized_in_class=True)
         add_field(Var('_field_defaults', dictype), is_initialized_in_class=True)
         add_field(Var('_source', strtype), is_initialized_in_class=True)
+        add_field(Var('__annotations__', ordereddictype), is_initialized_in_class=True)
```

Comment:
`ordereddictype` and `dictype` are actually the same thing here (they're initialized the same way). Maybe I should just have cleaned up `ordereddictype`.